Tests/DatadogTests/Datadog/Mocks/SystemFrameworks/FoundationMocks.swift

```diff
@@ -33,7 +33,15 @@ import Foundation
 
 // MARK: - Basic types
 
-extension Data {
+protocol AnyMockable {
+    static func mockAny() -> Self
+}
+
+protocol RandomMockable {
+    static func mockRandom() -> Self
+}
```

Comment:
Nice 💪! I feel that we could even push this further with some level of code generation in the future 💡.